shopelectro/models.py

```diff
@@ -295,3 +296,17 @@ def serialize_tags_to_title(tags: TagQuerySet) -> str:
         type_delimiter=settings.TAGS_TITLE_DELIMITER,
         group_delimiter=settings.TAG_GROUPS_TITLE_DELIMITER
     )
+
+
+class PageExcludeModelTManager(TreeManager):
+
+    def get_queryset(self):
+        return super().get_queryset().exclude(type=Page.MODEL_TYPE)
+
+
+class PageExcludeModelT(Page):
```

Comment:
`PageModelExcludedTManager` will be more clear.
Page don't exclude Model here. "Page" and "Model" should be together. `PageModel` is one entity

Reply:
@duker33 
>Page don't exclude Model here

I meant here Page model without ["MODEL_TYPE"](https://github.com/fidals/refarm-site/blob/master/pages/models.py#L74) pages, so maybe `ExcludedModelTPage` will be more appropriate?

Reply:
@artemiy312 , got you point. It might be good to start name with Page, but it maybe impossible.
So, choose name for your own plz :)

You may find interesting some variants:
`NotModelPage`, `PageWithNoModelT`, `PageWithoutModelT`